tests/test_packages/test_contracts/test_erc1155/test_contract.py

```diff
@@ -641,12 +641,48 @@ class TestCosmWasmContract(BaseContractTestCase):
     ledger_identifier = FetchAICrypto.identifier
     path_to_contract = Path(ROOT_DIR, "packages", "fetchai", "contracts", "erc1155")
     fund_from_faucet = True
+    deploy_tx_fee = 10000000000000000
+
+    @classmethod
+    def _deploy_contract(
```

Comment:
Why is this method added?

Reply:
it's an override mehod to add tx fee applied.